tutorial/01_image_filtering.md

```diff
@@ -57,26 +52,27 @@ slideshow:
 ---
 step_signal = np.zeros(100)
 step_signal[50:] = 1
+
 fig, ax = plt.subplots()
 ax.plot(step_signal)
 ax.margins(y=0.1)
 ```
 
 +++ {"slideshow": {"slide_type": "notes"}}
 
-Now add some noise to this signal:
+Next we add some noise to this signal:
 
 ```{code-cell} ipython3
 ---
 slideshow:
   slide_type: fragment
 ---
 # Just to make sure we all see the same results
-np.random.seed(0)
```

Comment:
Unfortunately both APIs give different results for the same "seed". `seed(0)` was actually a really good seed from a teaching perspective because the filters work really well while it still looks random enough. If found `default_rng(44)` to show similar results.